packages/docs/src/pages/en/features/theme.md

```diff
@@ -12,363 +12,199 @@ nav: Theme
+</template>
 
-export default new Vuetify({
-  theme: {
-    options: { themeCache },
-  },
-})
+<style>
+  .custom-class {
+    background: var(--v-theme-green)
+    color: var(--v-theme-on-green)
+  }
+</style>
 ```
 
-### Custom properties
-
-Enabling `customProperties` will also generate a [css variable](https://developer.mozilla.org/en-US/docs/Web/CSS/--*) for each theme color, which you can then use in your components' `<style>` blocks.
+## Color variations
 
-<alert type="warning">
-
-  Custom properties are not natively supported in Internet Explorer. Polyfills are available—**with some limitations**—for Internet Explorer 11:
-
-- [Custom properties polyfill](https://github.com/nuxodin/ie11CustomProperties)
-- [CSS vars polyfill](https://github.com/jhildenbiddle/css-vars-ponyfill)
-
-</alert>
+If you don't want to be bothered with coming up with lighten/darken variations of your colors, you can let Vuetify generate them for you using the **variations** property. The following configuration will generate 1 lighten variant and 2 darken variants for the `primary` and `secondary` colors.
 
 ```js
 // src/plugins/vuetify.js
+import { createApp } from 'vue'
+import { createVuetify } from 'vuetify'
 
-import Vue from 'vue'
-import Vuetify from 'vuetify/lib'
-
-export default new Vuetify({
+export default createVuetify({
   theme: {
-    options: { customProperties: true },
-  },
-})
-```
-
-```html
-<style scoped>
-  .something {
-    color: var(--v-primary-base);
-    background-color: var(--v-accent-lighten2);
+    defaultTheme: 'myCustomTheme',
+    variations: {
+      colors: ['primary', 'secondary'],
+      lighten: 1,
+      darken: 2,
+    },
+    themes: {
+      ...
+    }
   }
-</style>
+})
 ```
 
-### CSP nonce
-
-Pages with the `script-src` or `style-src` CSP rules enabled may require a **nonce** to be specified for embedded style tags.
-
 ```html
-<!-- Use with script-src -->
-Content-Security-Policy: script-src 'self' 'nonce-dQw4w9WgXcQ'
-
-<!-- Use with style-src -->
-Content-Security-Policy: style-src 'self' 'nonce-dQw4w9WgXcQ'
-```
+<template>
+  <div class="text-primary-lighten-1">text color</div>
 
-```js
-// src/plugins/vuetify.js
+  <div class="text-primary-darken-1">text color</div>
 
-import Vue from 'vue'
-import Vuetify from 'vuetify/lib'
-
-export default new Vuetify({
-  theme: {
-    options: { cspNonce: 'dQw4w9WgXcQ' },
-  },
-})
+  <div class="text-primary-darken-2">text color</div>
+</template>
 ```
 
-### Variations
+## Disable theme
 
-When Vuetify generates your *application's theme*, it creates **9 variants** for each color. For majority of users, these variants are rarely used. This is an **opt in** feature that will be __false by default__ in the next major version.
+The theme functionality can be disabled by setting the **theme** configuration property to `false`. This prevents the creation of the Vuetify stylesheet, and theme classes will not be applied to components.
```

Comment:
This is kind of confusing, could you help with rewording the first sentence into active voice?

Reply:
Confusing in what way?